_data/events.yaml

```diff
@@ -152,10 +152,10 @@ onetime:
   location: Online (Discord Server) <a href="https://github.com/UW-UPL/Talks/blob/master/archive/Lighting%20Talk%20%231.pdf">slides</a>
   description: 'In this talk I will cover my experiences using version control and other collaboration tools to write and deploy usable services. Topics will include Git, code review, tickets, deployment, and more! This will be geared towards an eager learner who has some knowledge of version control but no formal experience.'
 
-- title: 'Lightning Talk: To Be Announced (Michael Gira)'
+- title: 'Lightning Talk: Word Embeddings and Prejudice (Michael Gira)'
   date: 2021-04-14 7:30 PM
   location: Online (Discord Server)
-  description: ''
+  description: 'Let''s dive into the world of natural language processing and word embeddings! We will talk about their applications, quirks, and unfortunate prejudices. We''ll conclude with ways that researchers are trying to minimize prejudice and make these systems fair for everyone. No former experience necessary.'
```

Comment:
Are the double apostrophe's necessary/intentional in let's and We'll?

Reply:
Yeah, I did a cursory search and I believe that 2 single quotes are how you escape single quotes in a YAML string https://yaml.org/spec/current.html#id2534365